input/parameters/lithium-ion/cathodes/lico2_Marquis2019/lico2_data_example.csv

```diff
@@ -0,0 +1,50 @@
+0.000000000000000000e+00 4.714135898019971016e+00
```

Comment:
Maybe we should agree on a format for the data (like headers, where the data comes from...).